src/otx/core/model/detection.py

```diff
@@ -60,6 +61,29 @@ def __init__(
             tile_config=tile_config,
         )
 
+    def configure_metric(self) -> None:
+        """Configure the metric."""
+        if not callable(self.metric_callable):
+            raise TypeError(self.metric_callable)
+
+        metric = self.metric_callable(self.label_info)
+
+        if not isinstance(metric, (Metric, MetricCollection)):
+            msg = "Metric should be the instance of `torchmetrics.Metric` or `torchmetrics.MetricCollection`."
+            raise TypeError(msg, metric)
+
+        msg_fmeasure = (
+            "Detection should contain FMeasure computation to choose the best confidence threshold for testing."
+        )
+        if isinstance(metric, MetricCollection) and not any(isinstance(m, FMeasure) for m in metric.values()):
+            log.info(msg_fmeasure)
+            metric.add_metrics(FMeasure(self.label_info))
+        elif not isinstance(metric, FMeasure):
+            log.info(msg_fmeasure)
+            metric = MetricCollection([metric, FMeasure(self.label_info)])
```

Comment:
```suggestion
        msg = (
            "Detection should contain FMeasure computation to choose the best confidence threshold for testing."
        )
        if isinstance(metric, MetricCollection) and not any(isinstance(m, FMeasure) for m in metric.values()):
            log.warning(msg)
            metric.add_metrics(FMeasure(self.label_info))
        elif not isinstance(metric, FMeasure):
            log.warning(msg)
            metric = MetricCollection([metric, FMeasure(self.label_info)])
```
I hope this logging to be `warning` level and the message variable to be renamed `msg` for consistency with others.

Reply:
removed this runtime logic at 8733dcd4f1e380b8feb9e5e99d2c89624eb22b4c